course_project/7/ElonMuskHW/src/main/java/com/example/elonmuskhw/HibernateConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.example.elonmuskhw;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+public class HibernateConfig {
```

Comment:
You can extract this to a config subpackage (good practice to keep all configs there)